packages/plugin-core/src/commands/CapitalizeText.ts

```diff
@@ -0,0 +1,40 @@
+import { VSCodeUtils } from "../utils";
+import { window } from "vscode";
+import _ from "lodash";
+import { DENDRON_COMMANDS } from "../constants";
+import { BasicCommand } from "./base";
+
+type CommandOpts = {};
+
+type CommandInput = {};
+
+type CommandOutput = void;
+
+export class CapitalizeTextCommand extends BasicCommand<
+  CommandOpts,
+  CommandOutput
+> {
+  static key = DENDRON_COMMANDS.CAPITALIZE_TEXT.key;
+  async gatherInputs(): Promise<CommandInput | undefined> {
+    return {};
+  }
+  async execute() {
+    // Get active text editor
+    const textEditor = VSCodeUtils.getActiveTextEditor();
+    if (_.isUndefined(textEditor)) {
+      window.showErrorMessage("No active document found");
+      return;
+    }
+
+    await textEditor.edit((editBuilder) => {
+      textEditor.selections.forEach((selection) => {
+        // Get the text of the selection
+        const text = textEditor.document.getText(selection);
+        const capitalizedText = text.toUpperCase();
```

Comment:
what would happen for different locale based text when running this?

Reply:
The behavior may not be desired, and may not correctly transform the text depending on the language. One possible fix is to import `env` from `vscode` and refactor line 33 as:

```const capitalizedText = text.toLocaleUpperCase(env.language);```

 so that all languages are supported.

Reply:
One possible issue with this approach is if bilingual users attempt to use the command, as my understanding is that VSCode will prioritize their primary locale. However, this still supports a lot more users than the previous iteration.